src/scandeval/dataset_configs.py

```diff
@@ -863,6 +863,21 @@ def get_dataset_config(dataset_name: str) -> DatasetConfig:
     "grein.",
     num_few_shot_examples=1,
     max_generated_tokens=256,
+    unofficial=True,
```

Comment:
You've accidentally marked the wrong dataset as unofficial - this should be moved to `ICESUM_CONFIG` 🙂

Reply:
Oh, whoops 😅